File: SME0241 - BECD 2024/Introdução ao python/imc.py
# Faça um programa que calcule o IMC de uma pessoa, dada a sua massa m em Kg e sua altura a em metros:

# IMC = m/kg²

# Entrada:
# massa altura

# Saída:
# imc
# classificação

entrada = list(input().split())
massa = float(entrada[0])
altura = float(entrada[1])
imc = massa/(altura**2)
if imc < 18.5:
    clas = 'abaixo do peso'
elif 18.5 <= imc < 24.9:
    clas = 'saudável'
elif 24.9 <= imc < 29.9:
    clas = 'sobrepeso'
elif 29.9 <= imc < 34.9:
    clas = 'obesidade grau 1'
elif 34.9 <= imc < 39.9:
    clas = 'obesidade severa'
else: 
    clas = 'obesidade mórbida'
print(f'{imc:.2f}\n{clas}')

File: SSC0601 - Eng Comp 2025/Semana 1/imc.cpp
/*O programa deve calcular o Índice de Massa Corporal (IMC) de uma pessoa a partir do peso e altura fornecidos. O IMC é calculado pela fórmula: IMC = peso/altura²

Após calcular o IMC, o programa deve classificar o valor do IMC nas seguintes categorias:
    Abaixo de 18.5: Abaixo do peso
    De 18.5 a 24.99..: Peso normal
    De 25 a 29.999...: Sobrepeso
    De 30 ou mais: Obesidade
*/
#include <bits/stdc++.h>
using namespace std;

int main(){
    float altura, peso;

    cin >> peso >> altura;

    float imc = peso/(altura*altura);

    if (18.5 > imc){
        cout << "Abaixo do peso";
    } else if (18.5 <= imc && imc < 24.99){
        cout << "Peso normal"; 
    } else if (25 <= imc && imc < 29.99){
        cout << "Sobrepeso";
    } else{
        cout << "Obesidade";
    }
} 